crates/sui-core/src/authority_active.rs

```diff
@@ -378,9 +379,17 @@ where
                 let _ = join_handle.await;
             }
         }
+    }
+
+    async fn respawn_node_sync_process_impl(
```

Comment:
we should really rename this to node_follow process, but that's not your fault 😄

Reply:
will address in a separate PR